spotify_player/src/client/mod.rs

```diff
@@ -7,7 +7,10 @@ use crate::{
     event::{ClientRequest, PlayerRequest},
     state::*,
 };
-use anyhow::{Context as AnyhowContext, Result};
+
+#[cfg(any(feature = "lyric-finder", feature = "image"))]
+use anyhow::Context as AnyHowContext;
```

Comment:
```suggestion
use anyhow::Context as _;
```

I just realize we don't really to name the import here.

Reply:
Yeah, I think it's only a matter of getting it in the scope. I guess there is no need to rename it.